mmdet/core/evaluation/mean_ap.py

```diff
@@ -572,6 +573,9 @@ def eval_map(det_results,
             Default: False.
         use_group_of (bool): Whether to use group of when calculate TP and FP,
             which only used in OpenImages evaluation. Default: False.
+        mode (str): 'area' or '11points', 'area' means calculating the area
+            under precision-recall curve, '11points' means calculating the
+            average precision of recalls at [0, 0.1, ..., 1]. Default: 'area'.
```

Comment:
```suggestion
            average precision of recalls at [0, 0.1, ..., 1]. Defaults to 'area'.
```

Reply:
Change to `Defaults to` style? But at line 575 and others they use `Default:` style

Reply:
Yes, you can change all `Default:` to `Defaults to` in this file.

Reply:
We are going to unify using `defaults to` in mmdet

Reply:
I've changed all `"Default:"` to `"Defaults to"` in this file.